src/Model/Behavior/ProfferBehavior.php

```diff
@@ -101,7 +105,14 @@ public function beforeSave(Event $event, Entity $entity, ArrayObject $options, P
                             $imageTransform = new ImageTransform($this->_table, $path);
                         }
 
-                        $imageTransform->processThumbnails($settings);
+                        $thumbnailPaths = $imageTransform->processThumbnails($settings);
```

Comment:
Thank you for your reply!

I'm not sure, too.
Do you think it's better to fix to pass array?